Unreal Engine plugin documentation update (#11625)
getsentry/sentry-docs: platform-includes/performance/sampling-function-intro/unreal.mdx

To use the sampling function, set the <PlatformIdentifier name="traces-sampler" /> option in your SDK configuration to a function that will accept a <PlatformIdentifier name="sampling-context" /> dictionary and return a sample rate between 0 and 1. For example:

<PlatformContent includePath="performance/traces-sampler-as-sampler" />

<Alert level="info">

> The Unreal Engine SDK doesn't currently support sampling functions on Windows/Linux (<PlatformIdentifier name="traces-sampler" />).

Reviewer: ```suggestion
Currently there's no support for sampling functions on Windows/Linux (<PlatformIdentifier name="traces-sampler" />).
```
